rollup.config.mjs

```diff
@@ -15,13 +18,13 @@ export default {
     sourcemap: true,
     interop: false
   }],
-  external: [
-    'clsx',
-    'react',
-    'react-dom'
-  ],
+  external: id => !id.startsWith('.') && !isAbsolute(id),
```

Comment:
This will mark all non-relative imports as external, in other words all npm dependencies.